### DISCUSSION TITLE
FileUpload Drag and Drop not working on v14.

### DISCUSSION THREAD

**Discussion by TheMightyE** [primefaces/community#1915] [PrimeFaces] [answered]
Hello. Is anyone else having issues with FileUplod's drag and drop feature not working on PF v14? @melloware if you come across my post here, I tried on the [v14 showcase](https://quarkus-faces-melloware-8a6a34c1.koyeb.app/ui/file/upload/dnd.xhtml?jfwid=60464) you shared in a different post earlier and it's the same there where dragging a file into the component is not working. It works fine on [v13.0.10-SNAPSHOT showcase](https://www.primefaces.org/showcase/ui/file/upload/dnd.xhtml?jfwid=e7c54).

**Comment by melloware** [comment, marked as answer]
See: https://github.com/primefaces/primefaces/issues/11879

**Comment by melloware** [comment]
14.0.1 is on Maven Central now

**Reply by TheMightyE** [reply]
Thanks for the update! Really appreciate it.
